tests/src/hipEnvVarDriver.cpp

```diff
@@ -32,43 +32,59 @@ OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWA
 #include <thread>
 #include "test_common.h"
 
+
 using namespace std;
 
 int getDeviceNumber() {
     FILE* in;
     char buff[512];
-    string str;
+    string directed_dir = "directed_tests" + string(PATH_SEPERATOR_STR) + "hipEnvVar -c";
+    string dir = "." + string(PATH_SEPERATOR_STR) + "hipEnvVar -c";
```

Comment:
Make it global const

Reply:
What might be the purpose in making it global? I cannot use directed_dir and dir for both getDeviceNumber and getDevicePCIBusNumRemote as they have different arguments (namely, getDeviceNumber requires -c and getDevicePCIBusNumRemote needs -d)

Reply:
-c | -d the only difference, all the rest - copy/paste of string constants.